apps/slides/slides.md

```
@@ -0,0 +1,645 @@
+dragPos:
+square: 691,32,167,\_,-16
+
+---
+
+# Draggable Elements
+
+Double-click on the draggable elements to edit their positions.
+
+<br>
+
+###### Directive Usage
```

Markdownlint (0.37.0)

Expected: h2; Actual: h6
Heading levels should only increment by one level at a time

(MD001, heading-increment)